### PR TITLE
Download conversation message media content

### PR DESCRIPTION
**Pull request by tkanzakic**
**Checklist**
- [X] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)

### CODE REVIEW

File: packages/twilio/src/conversations/TwilioConversation.ts [hunk 1]
```diff
@@ -83,6 +85,19 @@ class TwilioConversation {
...
     return twilioWebhook.toJSON();
   }
+
+  async getMedia(mediaId: string) {
+    const { accountSid, authToken, serviceId } = this.config;
+    const url = `https://${accountSid}:${authToken}@mcs.us1.twilio.com/v1/Services/${serviceId}/Media/${mediaId}`;
+    let response = await fetch(url);
+    if (!response.ok) return undefined;
+    const media = await response.json();
+    const mediaUrl = media['links']['content_direct_temporary'];
```
Comment (guilean) [inline packages/twilio/src/conversations/TwilioConversation.ts:95]: Could it be that media is undefined or null at some point? This line would crash then.

Comment (guilean) [inline packages/twilio/src/conversations/TwilioConversation.ts:95]: Could it be that media is undefined or null at some point? This line would crash then.

File: packages/twilio/src/conversations/index.ts [hunk 2]
```diff
@@ -2,9 +2,10 @@ import TwilioConversation from './TwilioConversation';
 
 export const useTwilioConversation = (
   accountSid: string = process.env.TWILIO_ACCOUNT_SID,
-  authToken: string = process.env.TWILIO_AUTH_TOKEN
+  authToken: string = process.env.TWILIO_AUTH_TOKEN,
+  serviceId: string = process.env.TWILIO_CHAT_SERVICE_SID
 ) => {
-  return new TwilioConversation({ accountSid, authToken });
+  return new TwilioConversation({ accountSid, authToken, serviceId });
```
Comment (guilean) [inline packages/twilio/src/conversations/index.ts:8]: Simplify to `() => (new TwilioConversation({ accountSid, authToken, serviceId }))` ?

File: packages/twilio/src/sms/index.ts [hunk 3]
```diff
@@ -2,9 +2,10 @@ import { TwilioSms } from './TwilioSms';
 
 export const useTwilioSms = (
   accountSid = process.env.TWILIO_ACCOUNT_SID,
-  authToken = process.env.TWILIO_AUTH_TOKEN
+  authToken = process.env.TWILIO_AUTH_TOKEN,
+  serviceId = process.env.TWILIO_CHAT_SERVICE_SID
 ): TwilioSms => {
-  return new TwilioSms({ accountSid, authToken });
+  return new TwilioSms({ accountSid, authToken, serviceId });
```
Comment (guilean) [inline packages/twilio/src/sms/index.ts:8]: Ditto

File: packages/twilio/src/conversations/index.ts (hunk 2 above)
Comment (tkanzakic) [reply packages/twilio/src/conversations/index.ts]: done

File: packages/twilio/src/sms/index.ts (hunk 3 above)
Comment (tkanzakic) [reply packages/twilio/src/sms/index.ts]: done

File: packages/twilio/src/conversations/TwilioConversation.ts (hunk 1 above)
Comment (tkanzakic) [reply packages/twilio/src/conversations/TwilioConversation.ts:95]: it should not be undefined if response is ok, we can add a _defensive strategy_ if you prefer it

**Review by guilean — APPROVED** [review]

**Pull request merged**